source/Calamari.AzureAppService/Behaviors/TargetDiscoveryBehaviour.cs

```diff
@@ -27,9 +23,9 @@ namespace Calamari.AzureAppService.Behaviors
 {
     public class TargetDiscoveryBehaviour : IDeployBehaviour
     {
-        private const string WebAppSlotsType = "sites/slots";
-        private const string WebAppType = "sites";
-        private const int PageSize = 500;
+        private const string WebAppSlotsType = "microsoft.web/sites/slots";
+        private const string WebAppType = "microsoft.web/sites";
```

Comment:
It might be nice to have a short comment explaining what these values represent and where they came from.

Reply:
Yep good idea - now to find some doco explaining these values 😅